core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -771,7 +772,7 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
     verifyThreads("data-plane-kafka-socket-acceptor-", config.listeners.size)
 
     verifyProcessorMetrics()
-    verifyMarkPartitionsForTruncation()
+//    verifyMarkPartitionsForTruncation()
```

Comment:
so this is only applicable when JBOD feature is used?

Reply:
yes